### PR TITLE
apply sme updates

### LINKED ISSUE #1729: Edit to "View invoice history in the Cloud Office Control Panel"
https://github.com/rackerlabs/rackspace-how-to/blob/master/content/rackspace-email/view-invoice-history-cloud-office-control-panel.md

Current Section Without Changes: To view past invoices for your Cloud Office account, log in to the control panel and perform the following steps:

    At the top of the page, click your account name and select Billing & Payments.

    On the Billing & Payments tab, click Invoice History.

    A list of invoices appears, with the amount of each invoice and a brief description.

    To view a detailed invoice, click the invoice number.
    To print the invoice, click Print Invoice in the upper-right corner of the invoice window.
    To export the invoice as a CSV, click Export CSV file, and then save the .csv file to your computer.

*Proposed Changes:* To view past invoices for your Cloud Office account, log in to the Cloud Office Control Panel and perform the following steps:
1.	In the top right-hand corner of the page, click on your admin ID name/account number to activate a dropdown menu; then, select Billing & Payments.
2.	From the Billing & Payments tab, click Invoices.
A list of invoices appears, with the amount of each invoice and a brief description.
3.	To view a detailed invoice, click the invoice number.
a.	To print the invoice, click Print Invoice in the upper-right corner of the invoice window.
b.	To export the invoice as a CSV, click Export CSV, and then save the .csv file to your computer.

[To view past invoices for your Cloud Office account.docx](https://github.com/rackerlabs/rackspace-how-to/files/672242/To.view.past.invoices.for.your.Cloud.Office.account.docx)

### PR DESCRIPTION
**Pull request by stephamon**
fixes #1729 

@kmsholcomb can you give this a look?

### CODE REVIEW

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-187d55ae6e/how-to/).

**Comment by kmsholcomb** [comment]
Will be happy to, @stephamon! I'll probably do it first thing tomorrow.

**Comment by kmsholcomb** [comment]
Hi @stephamon. The attached doc has just a few nitpicky edits. Please let me know if you have any questions. Thanks!

[View invoice history in the Cloud Office Control Panel_editsKH.pdf](https://github.com/rackerlabs/rackspace-how-to/files/696762/View.invoice.history.in.the.Cloud.Office.Control.Panel_editsKH.pdf)
[View invoice history in the Cloud Office Control Panel_editsKH.docx](https://github.com/rackerlabs/rackspace-how-to/files/696763/View.invoice.history.in.the.Cloud.Office.Control.Panel_editsKH.docx)

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-131a663705/how-to/). :bow:

File: content/rackspace-email/view-invoice-history-cloud-office-control-panel.md [hunk 1]
```diff
@@ -5,21 +5,23 @@ title: View invoice history in the Cloud Office Control Panel
...
 ---
 
 To view past invoices for your Cloud Office account, [log in to the
-control panel](https://cp.rackspace.com/) and perform the following
+Cloud Office Control Panel](https://cp.rackspace.com/) and perform the following
 steps:
 
-1. At the top of the page, click your account name and select **Billing & Payments**.
-2. On the **Billing & Payments** tab, click **Invoice History**.
+1. In the top-right corner of the page, click on your admin ID name or account number and select **Billing & Payments** from the drop-down menu.
```
Comment (nimbinatus) [inline content/rackspace-email/view-invoice-history-cloud-office-control-panel.md]: *click on* should be *click*

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-b386f199ce/how-to/). :bow:

**Comment by nimbinatus** [comment]
LGTM

**Pull request merged**